odl/discr/lp_discr.py

```diff
@@ -212,23 +212,28 @@ def exponent(self):
         """The exponent ``p`` in ``L^p``."""
         return self._exponent
 
-    def element(self, inp=None):
+    def element(self, inp=None, **kwargs):
         """Create an element from ``inp`` or from scratch.
 
         Parameters
         ----------
         inp : `object`, optional
-            The input data to create an element from. Must be
-            recognizable by the `LinearSpace.element` method
-            of either `RawDiscretization.dspace` or
-            `RawDiscretization.uspace`.
+            The input data to create an element from. It needs to be
```

Comment:
This never had an example to start with, but I agree. I'll take the one from this conversation plus a bit.